src/AddItem/AddItem.tsx

```diff
@@ -1,8 +1,13 @@
-import { useState } from 'react';
+import { FC, useState } from 'react';
```

Comment:
it is not a mistake but in my opinion, code looks a bit cleaner when type imports are separated
```suggestion
import type { FC } from 'react';
import { useState } from 'react';
```

Reply:
It was corrected in this commit https://github.com/Sergey-Rogalskiy/intelligitodo/pull/9/commits/0fcfef6d890db47094080b93e97b97db85ca1ea3